packages/make-styles/src/runtime/resolveStyleRules.test.ts

```diff
@@ -586,8 +586,8 @@ describe('resolveStyleRules', () => {
   describe('output', () => {
     it('contains less members for properties that do not depend on text direction', () => {
       expect(resolveStyleRules({ color: 'red', paddingLeft: '10px' })).toEqual({
-        color: ['', 'fe3e8s90', '.fe3e8s90{color:red;}'],
-        paddingLeft: ['', 'frdkuqy0', '.frdkuqy0{padding-left:10px;}', '.rfrdkuqy0{padding-right:10px;}'],
+        color: ['', 'fe3e8s90', '.fe3e8s90{color:red;}', undefined, undefined],
```

Comment:
```suggestion
        color: ['', 'fe3e8s90', '.fe3e8s90{color:red;}'],
```

🚨 Please update implementation to match previous output, for LTR we should not have `undefined` there.